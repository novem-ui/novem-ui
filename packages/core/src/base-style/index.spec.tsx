import React from 'react'

import withTheme from '@novem-ui/theme/src/utils/with-theme'
import { render } from '@testing-library/react'

import BaseStyle from '.'

describe('<BaseStyle />', () => {
  test('should match snapshot (default)', () => {
    const Component = () => <BaseStyle />
    const WrappedComponent = withTheme(Component)
    const { container } = render(<WrappedComponent />)

    expect(container).toMatchSnapshot()
  })
})
