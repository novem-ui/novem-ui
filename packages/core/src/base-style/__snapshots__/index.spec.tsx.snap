// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseStyle /> should match snapshot (default) 1`] = `<div />`;
